packages/schemas/src/Components/Concerns/HasState.php

```diff
@@ -68,6 +68,8 @@ trait HasState
      * @var array<string>
      */
     protected array $cachedStripCharacters;
+	
+	protected bool | Closure $isTrimmed = false;
```

Comment:
Would you also want to cache the result of `isTrimmed`, similar to `cachedStripCharacters`?

Reply:
Yeah I think that would be good